.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,54 @@
+name: aws-deploy
+
+on:
+  pull_request:
+    branches: '*'
+  push:
+    branches: [prod]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  sceptre-deploy:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    needs: pre-commit
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Cache Pipenv virtualenv
+        uses: actions/cache@v2
+        id: pipenv-cache
+        with:
+          path: ~/.local/share/virtualenvs
+          key: ${{ runner.os }}-pipenv-v2-${{ hashFiles('**/Pipfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pipenv-v2-
+      - name: Install Dependencies
+        uses: VaultVulp/action-pipenv@v2.0.1
+        if: steps.pipenv-cache.outputs.cache-hit != 'true'
+        env:
+          PIPENV_NOSPIN: 'true'
+          WORKON_HOME: ~/.local/share/virtualenvs
+          PIPENV_CACHE_DIR: ~/.local/share/pipcache
+        with:
+          command: install --dev
+      - name: Login Using CI Service User
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.NEXTFLOW_PROD_CI_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Deploy to AWS Account
+        uses: VaultVulp/action-pipenv@v2.0.1
+        with:
+          command: run sceptre launch prod --yes
```

Comment:
ahh, cool. i just noticed this.. https://github.com/marketplace/actions/sceptre-github-action

Reply:
That's cool too -- going to start using that one and the pipenv one

Reply:
Another good find! This actually streamlines the CI workflow considerably.